src/Classes/PassiveTree.lua

```diff
@@ -572,54 +438,67 @@ local PassiveTreeClass = newClass("PassiveTree", function(self, treeVersion)
 		end
 	end
 
-	-- Build ModList for legion jewels
-	for _, node in pairs(self.legion.nodes) do
-		-- Determine node type
-		if node.m then
-			node.type = "Mastery"
-		elseif node.ks then
-			node.type = "Keystone"
-			if not self.keystoneMap[node.dn] then -- Don't override good tree data with legacy keystones
-				self.keystoneMap[node.dn] = node
+	self.legion = main.tree.legion
+	if not self.legion then
+		self.legion = LoadModule("Data/TimelessJewelData/LegionPassives")
+
```

Comment:
Is this logic just for being able to skip these loops if the legion nodes have already been loaded?